example/src/Demo.jsx

```diff
@@ -67,7 +67,7 @@ class Demo extends React.Component {
               }}
               flags={{ correctnewlines: false }}
               suggestedEdits
-              oauthUrl={this.props.oauthUrl}
+              oauth={this.props.oauth}
```

Comment:
This isn't a change from the new API key, is it?

Reply:
I recently renamed `oauthUrl` to `oauth` in the ApiExplorer props, but this was still using the old way.